### DISCUSSION TITLE
Title: DAI 2024-2025 - My picocli project - Thomas Stäheli

### DISCUSSION THREAD

**Discussion by thomasstaheli** [heig-vd-dai-course/heig-vd-dai-course#319] [Show and tell]
[](https://github.com/thomasstaheli/DAI_labo1_Thomas_Staheli)

**Comment by thomasstaheli** [comment]
https://github.com/thomasstaheli/DAI_labo1_Thomas_Staheli

**Reply by ludelafo** [reply]
It seems I cannot access the repository, is it private? Please make it open or invite me to it. :)
